helpers/init.ts

```diff
@@ -3,12 +3,40 @@ import { HardhatRuntimeEnvironment } from 'hardhat/types'
 
 import { RuntimeConfig } from './types/common'
 
-export default async function init(hre?: HardhatRuntimeEnvironment): Promise<RuntimeConfig> {
-  const provider = hre ? hre.ethers.provider : (await import('hardhat')).ethers.provider
-  // const provider = new ethers.providers.JsonRpcProvider('http://localhost:8545', 2137)
-  const signer = provider.getSigner(0)
+export async function impersonate(
+  provider: providers.JsonRpcProvider,
+  impersonatedAddress: string,
+) {
+  await provider.send('hardhat_impersonateAccount', [impersonatedAddress])
+
+  const signer = provider.getSigner(impersonatedAddress)
   const address = await signer.getAddress()
 
+  return { signer, address }
+}
+
+export async function impersonateRichAccount(provider: providers.JsonRpcProvider) {
+  // https://etherscan.io/address/0xe3dd3914ab28bb552d41b8dfe607355de4c37a51
+  const richAccount = '0xe3dd3914ab28bb552d41b8dfe607355de4c37a51'
+
+  return await impersonate(provider, richAccount)
+}
```

Comment:
Why you need this rich account? Test account have 1000ETH isn't it enough? If you need other tokens I would use Uniswap or other exchange to get them.

Reply:
We don't. You're right. I added it for convenience when I was fighting with some other issues - then it got baked in. That account owns lots of different ERC20s in quantity.

I can refactor it and replace with Swap.